src/Hypergrid/events.js

```diff
@@ -560,7 +560,7 @@ exports.mixin = {
                  */
                 function(mouseEvent) {
                 var isMouseDownCell = this.mouseDownState && this.mouseDownState.gridCell.equals(mouseEvent.gridCell);
-                if (isMouseDownCell && mouseEvent.mousePointInClickRect) {
+                if (isMouseDownCell) {
```

Comment:
mousePointInClickRect is now replaced by mousePointInLeftClickRect and mousePointInRightClickRect.

And we nolonger check for the fact that it is clicked.
Instead we pass the click result down stream and let the client decide what to do.